recipes/openssl/1.x.x/conanfile.py

```diff
@@ -678,6 +678,7 @@ def package(self):
     def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "OpenSSL"
         self.cpp_info.names["cmake_find_package_multi"] = "OpenSSL"
+        self.env_info.OPENSSL_ROOT_DIR = self.package_folder
```

Comment:
The problem is that CMake automatically sets <PROJECT>_ROOT_DIR. If you need to do it for OpenSSL, you also need for any project. I think it's a problem how to use conan virtualenv generator and not a recipe bug.